src/shim.js

```diff
@@ -91,23 +91,39 @@ const commands = {
         completeCommand(command);
     },
 
+    loadPageContent: function (command) {
+        let page = objectSpace[command.target];
+        if(!page) {
+            command.response = false;
+            return completeCommand(command);
+        }
+
+        page.onLoadFinished = result => {
+            page.onLoadFinished = null;
```

Comment:
Yes it does. Good point. But I guess if they are not using `loadContent` and a new page everytimg then it wouldn't matter. But this is a bug.